base/applications/drwtsn32/sysinfo.cpp

```diff
@@ -9,6 +9,15 @@
 #include <winreg.h>
 #include <reactos/buildno.h>
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include <udmihelp.h>
+#include <dmilib.h>
+#ifdef __cplusplus
+}
+#endif
```

Comment:
It may be interesting to have these headers using the `extern "C"` block (as done on windows.h for example).